src/app/content/components/Page/PageToasts.tsx

```diff
@@ -4,32 +4,37 @@ import styled, { css } from 'styled-components/macro';
 import {
   bookBannerDesktopMiniHeight,
   bookBannerMobileMiniHeight,
-  toolbarDesktopHeight,
-  toolbarMobileHeight,
-  toolbarMobileSearchWrapperHeight
+  toolbarMobileSearchWrapperHeight,
+  toolbarWidth,
+  topbarDesktopHeight,
+  topbarMobileHeight
 } from '../../../content/components/constants';
 import ToastNotifications from '../../../notifications/components/ToastNotifications';
 import { groupedToastNotifications } from '../../../notifications/selectors';
 import theme from '../../../theme';
 import { mobileToolbarOpen as mobileToolbarOpenSelector } from '../../search/selectors';
+import { isVerticalNavOpenConnector } from '../utils/sidebar';
 
-export const desktopSearchFailureTop = bookBannerDesktopMiniHeight + toolbarDesktopHeight;
+export const desktopSearchFailureTop = bookBannerDesktopMiniHeight + topbarDesktopHeight;
 export const getMobileSearchFailureTop = ({mobileToolbarOpen}: {mobileToolbarOpen: boolean}) => mobileToolbarOpen
-  ? bookBannerMobileMiniHeight + toolbarMobileHeight + toolbarMobileSearchWrapperHeight
-  : bookBannerMobileMiniHeight + toolbarMobileHeight;
+  ? bookBannerMobileMiniHeight + topbarMobileHeight + toolbarMobileSearchWrapperHeight
+  : bookBannerMobileMiniHeight + topbarMobileHeight;
 
 // tslint:disable-next-line:variable-name
-export const ToastContainerWrapper = styled.div`
-  width: 100%;
+export const ToastContainerWrapper = isVerticalNavOpenConnector(styled.div`
```

Comment:
@TomWoodward this all works unless the page is reloaded with a sidebar open because in that case `content.tocOpen` is null. Is there a way around that?

Reply:
this component doesn't even seem to be using any of the properties provided by this conector, am i missing something?